sources/europe/hr/dgu-dof-2019-s-e.geojson

```diff
@@ -0,0 +1,656 @@
+{
+    "type": "Feature",
+    "properties": {
+        "category": "photo",
+        "id": "dgu-dof-2019-s-e",
+        "name": "dgu.hr: Croatia 2019 South+East Aerial imagery",
+        "icon": "https://osmlab.github.io/editor-layer-index/sources/europe/hr/dgu.png",
+        "start_date": "2019",
+        "end_date": "2019",
+        "url": "https://geoportal.dgu.hr/services/inspire/orthophoto_2019/ows?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=OI.OrthoimageCoverage&STYLES=&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
+        "permission_osm": "explicit",
+        "license_url": "https://narodne-novine.nn.hr/clanci/sluzbeni/2013_02_25_403.html",
```

Comment:
Is there a prior discussion somewhere about why this law permits use? Sorry I don't have the time to read through a whole piece of legislation.

Reply:
It's the same as in:
https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/hr/dgu-topo-25.geojson
https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/hr/dgu-topo-25-new.geojson
https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/hr/dgu-topo-200.geojson
https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/hr/dgu-topo-100.geojson
https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/hr/dgu-hok.geojson
https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/hr/dgu-dof-2018-w.geojson
https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/hr/dgu-dof-2017-s-e.geojson
https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/hr/dgu-dof-2014-2016.geojson
https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/hr/dgu-dof-2011.geojson

It's based on european freedom of information act.